### PR TITLE
Claims status layout

### PR DESCRIPTION
**Pull request by molindoz**
This PR will finish the layout work for:
• https://github.com/department-of-veterans-affairs/sunsets-team/issues/46
• https://github.com/department-of-veterans-affairs/sunsets-team/issues/47
• https://github.com/department-of-veterans-affairs/sunsets-team/issues/49
• https://github.com/department-of-veterans-affairs/sunsets-team/issues/72
• https://github.com/department-of-veterans-affairs/sunsets-team/issues/91

Notes:
• This does not close any of these issues as we still need to connect data from the api.
• I will also be adding screenshots for desktop/mobile screens

### CODE REVIEW

**Comment by cagedcrown** [comment]
## Screenshots
screen shot 2016-10-07 at 11 27 50 am

screen shot 2016-10-07 at 11 34 20 am

screen shot 2016-10-07 at 11 44 01 am

screen shot 2016-10-07 at 11 28 18 am

screen shot 2016-10-07 at 11 45 52 am

screen shot 2016-10-07 at 11 28 46 am

screen shot 2016-10-07 at 11 27 16 am

**Comment by cagedcrown** [comment]
Hey @jbalboni, can you review this PR when you get the chance?

File: src/sass/disability-benefits.scss [hunk 1]
```diff
@@ -129,18 +129,24 @@
...
   }
 }
 
-.you-have-no-claims {
+.ask-va-to-decide,
+.claim-decision-is-ready,
+.you-have-no-claims,
+.we-are-adding-your-details,
+.agreement {
+  background-color: $color-primary-alt-lightest;
```
Comment (jbalboni) [inline src/sass/disability-benefits.scss]: There's an AlertBox component that might work for these, or using the `usa-alert` classes (there are examples of both in edu benefits).

File: src/sass/disability-benefits.scss [hunk 2]
```diff
@@ -337,3 +347,31 @@
  }
 
 // TODO: Add media queries for responsiveness
+@media only screen {
```
Comment (jbalboni) [inline src/sass/disability-benefits.scss]: Is this for handling print layouts?

File: src/js/disability-benefits/components/TabNav.jsx [hunk 3]
```diff
@@ -5,15 +5,11 @@ import TabItem from './TabItem';
...
-      <div className="row">
-        <div className="medium-8 columns">
-          <ul className="va-tabs">
-            <TabItem tabpath="/your-claims/status" title="Status"/>
-            <TabItem tabpath="/your-claims/files" title="Files"/>
-            <TabItem tabpath="/your-claims/details" title="Details"/>
-          </ul>
-        </div>
-      </div>
+      <ul className="va-tabs">
```
Comment (jbalboni) [inline src/js/disability-benefits/components/TabNav.jsx]: Tab nav seems have some spacing gaps (compared to rx):

screen shot 2016-10-07 at 1 29 58 pm

**Review by jbalboni — APPROVED** [review]
This looks good, but I would look into making the different "boxes" above the subway map use our common AlertBox component or at least the styles it's using.

File: src/sass/disability-benefits.scss (hunk 1 above)
Comment (cagedcrown) [inline src/sass/disability-benefits.scss]: So `usa-alert` uses a different background color, so I went ahead and refactored the alert messages to use the new `claims-alert` class.

Comment (jbalboni) [inline src/sass/disability-benefits.scss]: Is it a slightly different blue? It looks the same as `usa-alert usa-alert-info`.

Comment (cagedcrown) [inline src/sass/disability-benefits.scss]: Hmm oh okay, yeah `usa-alert` showed up as a light gray color. And the info-icon from `usa-alert-info` wasn't in the design mocks that was sent to me.

Comment (jbalboni) [inline src/sass/disability-benefits.scss]: Yeah. I would just make a class to hide the icon and use as much of the usa-alert-* styling as you can.

Comment (cagedcrown) [inline src/sass/disability-benefits.scss]: Yeah that makes sense 👍

**Pull request merged**